### DISCUSSION TITLE
Ipad SidebarListStyle support

### DISCUSSION THREAD

**Discussion by Arneoldenhave** [frzi/swiftui-router#54] [Q&A]
Hi!

I'm struggling a little implementing routing within a List that has SidebarListStyle.

The results are quitte messy.

Is there support for this king of navigation? 

Thanks!

Arne

**Comment by frzi** [comment]
The example [TabViewRouting](https://github.com/frzi/SwiftUIRouter-Examples/tree/main/TabViewRouting) demonstrates how you can use SwiftUI Router with a `TabView`. The technique shouldn't be too different from using `List`'s [`init(selection:content:)`](https://developer.apple.com/documentation/swiftui/list/init(selection:content:)-1pisz)
